lib/rubocop/cop/mixin/surrounding_space.rb

```diff
@@ -4,11 +4,14 @@ module RuboCop
   module Cop
     # Common functionality for checking surrounding space.
     module SurroundingSpace
-      def space_between?(t1, _t2)
-        # Check if the range between the tokens starts with a space. It can
-        # contain other characters, e.g. a unary plus, but it must start with
-        # space.
-        t1.pos.source_buffer.source.match(/\G\s/, t1.pos.end_pos)
```

Comment:
While this was named `space_between?`, you can see that the second arg is not used. What was actually getting calculated here is if there is a `space_after` the token provided. If so, the method return a MatchData obj of " ".  Otherwise, it returns nil.

It was hard to move forward with a method with a deceiving name, as these cops are focussed on detecting and manipulating `space`.  So I renamed the method and fixed everywhere it was used accordingly. I've tried to squash this refactoring into one commit.

Below is a `space_before?` method that I use in these new cops, but `space_before` was not injected into any other cop.

Reply:
Seems like a good change to me. 👍